packages/fxa-settings/src/lib/metrics.ts

```diff
@@ -51,7 +52,7 @@ type ConfigurableProperties = {
   utm_term: Optional<string>;
 };
 
-type EventData = FlowQueryParams &
+type EventData = QueryParams &
```

Comment:
If any of the naming between `QueryParams` and `FlowQueryParams` feels awk let me know, I tried to minimize the amount of `FlowQueryParam` references I needed to change since we'll ultimately end up changing them back after we no longer need `showNewReactApp`.